src/thing/Portal.cpp

```diff
@@ -0,0 +1,112 @@
+#include "thing/Portal.h"
```

Comment:
Unless there are issues with what is presented here, let's get this checked in and I will evaluate the WiFiManager this friday and let you know what I think.